### PR TITLE
Feat/codelab basics 1

### PR DESCRIPTION
**Pull request by LarsDenBakker**
You can preview it at: https://deploy-preview-878--open-wc-org.netlify.com/codelabs/basics/1-web-component.html#0

TODO:

- [ ] Add further reading page

### CODE REVIEW

File: packages/codelabs/src/basics/1-web-component.md [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+  <body>
+    <script type="module">
+      import { LitElement } from 'https://unpkg.com/lit-element?module';
+
+    </script>
+  </body>
+</html>
+```
+
+Make sure you add type="module" to the script tag!
```
Comment (thepassle) [inline packages/codelabs/src/basics/1-web-component.md]:
```suggestion
Make sure you add `type="module"` to the script tag!
```

File: packages/codelabs/src/basics/1-web-component.md [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+
+Next, we need to define our class. Instead of extending `HTMLElement`, we are now extending `LitElement`. `LitElement` extends `HTMLElement` already, so we're still writing an actual web component:
+
+```js
+class WebComponentApis extends LitElement {
+  connectedCallback() {
+    // make
```
Comment (thepassle) [inline packages/codelabs/src/basics/1-web-component.md]: make?

File: packages/codelabs/src/basics/1-web-component.md [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+}
+
+customElements.define('web-component-apis', WebComponentApis);
+```
+
+```html
+  <web-component-apis></web-component-apis>
+```
+
+If you run this in the browser you should see `hello world` logged to the terminal.
```
Comment (thepassle) [inline packages/codelabs/src/basics/1-web-component.md]:
```suggestion
If you run this in the browser you should see `lit element connected` logged to the terminal.
```

File: packages/codelabs/src/basics/1-web-component.md [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+  </body>
+</html>
+```
+</details>
+
+## Web Components in the wild
+Web Components are being used in the wild by many projects and companies. Some examples:
+
+### Github
+Github uses web component for various part of their website. They're using just the Custom Elements API, relying on global styling. They use them as a progressive enhancements, on browsers without support or when javascript is turned off, there is a fallback text that is dispalyed.
```
Comment (thepassle) [inline packages/codelabs/src/basics/1-web-component.md]:
```suggestion
Github uses web component for various part of their website. They're using just the Custom Elements API, relying on global styling. They use them as a progressive enhancements, on browsers without support or when javascript is turned off, there is a fallback text that is displayed.
```

File: packages/codelabs/src/basics/1-web-component.md (hunk 2 above)
Comment (LarsDenBakker) [reply packages/codelabs/src/basics/1-web-component.md]: woops, will remove that one

**Pull request merged**